packages/server/graphql/graphql.ts

```diff
@@ -18,6 +18,7 @@ export interface GQLContext {
 export interface InternalContext {
   dataLoader: DataLoaderWorker
   authToken: AuthToken
+  ip: string
```

Comment:
I can see that I am able to get my IP from the InternalContext, so I thought I'd add it here. Alternatively, I could make the `ip` arg optional in https://github.com/ParabolInc/parabol/blob/6f207a89341461d03180a8cd157d424e426c7155/packages/server/graphql/private/mutations/removeAuthIdentity.ts#L65

Reply:
since this is on the private schema, the ip is gonna be a sales person. i'd hate to rate limit them, but i don't think it'll be a problem either way, so this is probably OK